gcloud-java-compute/src/main/java/com/google/gcloud/compute/DeprecationStatus.java

```diff
@@ -28,12 +30,13 @@
 /**
  * The deprecation status associated to a Google Compute Engine resource.
  *
- * @param <T> The Google Compute Engine resource to which the deprecation status refers to.
+ * @param <T> The Google Compute Engine resource identity to which the deprecation status refers to
```

Comment:
Nit: can remove the "to" at the end of the line b/c you have "to which" already.  Also applies to the Builder down below.